src/index.js

```diff
@@ -9,6 +9,17 @@ var writeFile = promisify(fs.writeFile);
 import parseChangelog from './parser';
 import buildChangelog from './builder'
 
+export function init (path) {
+  path = path || 'CHANGELOG.md';
+
+  return parse(
+    [
+      '# Change Log',
+      'All notable changes to this project will be documented in this file.'
+    ].join('\n')
```

Comment:
It's injected by default. See the `\n` at the end of this line https://github.com/contentful-labs/keepachangelog/pull/13/files#diff-c1129c8b045390789fa8ff62f2c6b4a9R34

But I'll an explicit empty line to make it more clear.